db/db_tunables.h

```diff
@@ -1261,6 +1261,9 @@ REGISTER_TUNABLE(
     "Deliberately allow insertion without constraint check to debug db_verify",
     TUNABLE_BOOLEAN, &gbl_debug_skip_constraintscheck_on_insert, INTERNAL, NULL,
     NULL, NULL, NULL);
+REGISTER_TUNABLE("debug.eventlog_rollat_size",
```

Comment:
move this to [debug_switches](https://github.com/bloomberg/comdb2/blob/master/util/debug_switches.c)?

Reply:
Hmm, we have way too many ways of setting debug parameters. Will do, thanks.

Reply:
Removed it totally. Thanks for suggestion.